python/grass/jupyter/display.py

```diff
@@ -31,18 +31,24 @@ def __init__(
         else:
             self._env = os.environ.copy()
 
+        # Make temporary folder for all our files
+        self._tmp_dir = Path("./tmp/")
+        try:
+            os.mkdir(self._tmp_dir)
+        except FileExistsError:
+            pass
```

Comment:
If user passes _filename_, they likely expect that the file will be created somewhere they desire and that it is a file they will use later. Otherwise there would be no reason to provide filename in the first place.